src/access/AccessControl.cpp

```diff
@@ -608,6 +609,10 @@ bool AccessControl::IsValid(const Entry & entry)
     return true;
 
 exit:
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(DataManagement, "AccessControl: %s %" CHIP_ERROR_FORMAT, log, err.Format());
+    }
     ChipLogError(DataManagement, "AccessControl: %s", log);
```

Comment:
This log is not redundant. The previous code was correct, but it didn't log details of an error.

Reply:
Er, right, I assume you mean "now redundant".

We can still end up here when `err == CHIP_NO_ERROR` and need to log things.  I will fix that up.

Reply:
Yes, "now redundant" :)